src/modules/site-settings/edit-site-details.tsx

```diff
@@ -308,6 +308,7 @@ export default function EditSiteDetails( { currentWpVersion, onSave }: EditSiteD
 											id="enable-https"
 											checked={ enableHttps }
 											onChange={ ( e ) => setEnableHttps( e.target.checked ) }
+											disabled={ isEditingSite }
```

Comment:
This is a fix for the HTTPS field not being disabled during save, correct?

Reply:
Thanks, I forgot to comment on that change. You are correct 👍